src/scripts/pokemons/RoamingPokemonList.ts

```diff
@@ -4,31 +4,39 @@
 ///<reference path="../../declarations/requirements/OneFromManyRequirement.d.ts"/>
 
 class RoamingPokemonList {
-    public static list: Partial<Record<GameConstants.Region, Array<RoamingPokemon>>> = {};
-    public static increasedChanceRoute: Array<KnockoutObservable<RegionRoute>> = new Array(GameHelper.enumLength(GameConstants.Region)).fill(0).map((route, region) => ko.observable(null));
+    public static list: Partial<Record<GameConstants.Region, Array<Array<RoamingPokemon>>>> = {};
+    // TODO: Right now increasedChanceRoute limits our subRegionGroups to 3.
+    // This is hardcoded because we cannot get number of subregiongroups at this point in the code.
+    public static increasedChanceRoute: Array<Array<KnockoutObservable<RegionRoute>>> = new Array(GameHelper.enumLength(GameConstants.Region))
+        .fill(0).map(() => new Array(3).fill(0).map(() => ko.observable(undefined)));
```

Comment:
Would it be worth limiting it to `4` now rather than later, as Alola will eventually be 4 sub regions, and chances are this piece of code will be forgotten when splitting it.